README.md

```diff
@@ -86,13 +86,10 @@ You'll need to do three things:
 * Install PhantomJS
 * Set up your environment:
 
-  Easiest thing is to create a `.env` like so:
+  Easiest thing is to create a `.env.development` file by running:
 
   ```
-  RESQUE_BRAIN_INSTANCES=localhost
-  RESQUE_BRAIN_INSTANCES_localhost=redis://localhost:6379
-  HTTP_AUTH_USERNAME=a
-  HTTP_AUTH_PASSWORD=a
+  > ruby bin/setup
```

Comment:
I think if the permissions in the repo are correct (via `chmod 755 bin/setup`), they should just be able to type `bin/setup` without the interpreter.